Examples/NoModules/Chapter 13/Ex13_12B/Ex13_12B.cpp

```diff
@@ -0,0 +1,21 @@
+// Sneak preview: use copy-and-swap for copy assignment operator
```

Comment:
My draft text does not have an Ex13_12B. Maybe you added it in the final text?

Reply:
The example is there, just not named. I'll add a comment.